include/units.h

```diff
@@ -993,10 +941,9 @@ namespace units
 	namespace traits
 	{
 		/**
-		* @brief		Trait which returns the `dimension` type that a unit is originally derived from.
-		* @details		Since units can be derived from other `unit` types in addition to `dimension` types,
-		*				the `dimension_type` typedef will not always be a `dimension` (or unit dimension).
-		*				Since compatible

```

Comment:
Did you want to say something on this line? It was incomplete so I got rid of it.